docker/Dockerfile.nvidia

```diff
@@ -40,34 +8,30 @@ ENV DEBIAN_FRONTEND noninteractive
   
 # nodesource: nvdashboard requires nodejs>=10  
 RUN echo 'deb [trusted=yes] https://developer.download.nvidia.com/hpc-sdk/ubuntu/amd64 /' > /etc/apt/sources.list.d/nvhpc.list
-RUN apt-get update -y && \
-    apt-get install -y -q \
-        apt-utils curl wget
+RUN apt-get update -y && apt-get install -y -q apt-utils curl wget vim
 
 RUN curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
     apt-get install -y -q \
-#        nvhpc-22-5-cuda-multi \
-        nvhpc-22-5 \
-        liblapack-dev  \
-        libblas-dev \
+        liblapack-dev libblas-dev \
         libibverbs-dev libmlx4-1 libmlx5-1 ibutils \
-# Devito Jupyter Notebooks and Ux experience
-        nodejs \
-        texlive-latex-extra texlive-fonts-recommended dvipng cm-super \
-        ffmpeg vim && \
-    wget -q -P /app/nvcomp_exts/ \
-        http://developer.download.nvidia.com/compute/nvcomp/2.2/local_installers/nvcomp_exts_x86_64_ubuntu18.04-2.2.tar.gz && \
+        nodejs ffmpeg \
+        texlive-latex-extra texlive-fonts-recommended dvipng cm-super
+
+# Install latest available nvhpc
+RUN ver=$(apt-cache show nvhpc | grep Source | awk '{print $2}') && apt-get install -y -q ${ver}
+
+RUN wget -q -P /app/nvcomp_exts/ http://developer.download.nvidia.com/compute/nvcomp/2.2/local_installers/nvcomp_exts_x86_64_ubuntu18.04-2.2.tar.gz && \
```

Comment:
I think this is only for compression which we don't support here in open source territory, so it probably can be dropped

Reply:
(@kenhester )

Reply:
doesn't hurt to have it it's tiny compared to the rest

Reply:
sure, but each unnecessary line of code is a grain of complexity we can avoid